internal/client/integrations/overrides.go

```diff
@@ -101,6 +102,7 @@ func mergeOverrides(eversion integrationVersionExternal, o overrides) (integrati
 					}
 					trigger.TriggerId = pubsubTrigger + *triggerOverride.ProjectId + "_" + *triggerOverride.TopicName
 					trigger.Properties["Subscription name"] = *triggerOverride.ProjectId + "_" + *triggerOverride.TopicName
+					trigger.Properties["Service account"] = *triggerOverride.ServiceAccount
```

Comment:
You will have to also override the IP Project name for this to work successfully